linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_files_permissions/ansible/shared.yml

```diff
@@ -0,0 +1,39 @@
+# platform = Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_sle
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = medium
+
+- rsyslog_etc_config="/etc/rsyslog.conf"
+{{% if product in ["debian9", "debian10", "debian11", "ubuntu1604", "ubuntu1804", "ubuntu2004", "sle15", "sle12"] %}}
+- desired_perm_mode="640"
+{{% else %}}
+- desired_perm_mode="600"
+{{% endif %}}
```

Comment:
There is an Ansible syntax error here.
I guess you wanted to specify a `set_fact`?

Reply:
Many thanks for the feedback :bow:. Not only the set_fact, but also other stuff I had missed in the ansible remediation, thanks to using different environment for ansible sandbox had missed couple of things. Should be ok now in [ea5b14e](https://github.com/ComplianceAsCode/content/pull/8882/commits/ea5b14e97b9f11d25990e8bd9e37d8fd468e257f)

Reply:
Thank you for the Ansible remediaiton!